workflow/metrics/metrics.go

```diff
@@ -153,6 +155,23 @@ func (m *Metrics) StopRealtimeMetricsForKey(key string) {
 		return
 	}
 
+	realtimeMetrics := m.workflows[key]
+	for _, metric := range realtimeMetrics {
+		if realtimeMetric, ok := m.customMetrics[metric]; ok {
+			realtimeMetric.completed = true
```

Comment:
Should this also update `lastUpdated` so that the metric lives TTL seconds beyond the end of the workflow rather than potentially ending immediately? It feels likely this would have been set recently, but this would ensure it.

Reply:
`lastUpdated` of the metric has been updated in workflow operation, so it is no need to update it again in `persistUpdates`.

Reply:
Ok.